README.md

```diff
@@ -151,7 +152,7 @@ Url
 Type --help for more information.
 ```
 
-**TIP**: Make sure to use the proper versions of Apache Spark 2.4 (or 2.3), Scala 2.11, and Java 1.8.0.
+**TIP**: Make sure to use the proper versions of Java 1.8.0, and Apache Spark 3.0 with Scala 2.12 or Apache Spark 2.4 (or 2.3) with Scala 2.11.
```

Comment:
FYI: [Spark 3 works with Java 11](http://spark.apache.org/docs/latest/#downloading) which gives better performance and is a supported Java version.

Reply:
Thanks for the info, will be worth looking into.